### DISCUSSION TITLE
Switch Default Color

### DISCUSSION THREAD

**Discussion by kg-lee** [chakra-ui/chakra-ui#6140] [General]
I am aware that we could use `colorScheme` with `Switch` to change the background color when the toggle is on. However, it seems that you cannot change the color when the toggle is off. 

Is there a way to achieve this?

**Comment by jTanG0506** [comment]
```
<Switch sx={{ 'span.chakra-switch__track:not([data-checked])': { backgroundColor: 'pink.300' } }} />
```

**Reply by chimejuliet** [reply]
Thank you for your response jTanG0506.

Please how can I change the switch color from white to any other color when the toggle is on or off

**Comment by asvinp** [comment]
I used the following to change the bg color while enabled (without colorscheme) in case someone's looking for it:

`sx={{
                        '.chakra-switch__track[data-checked]:not([data-theme])':
                          {
                            backgroundColor: 'lightblue',
                          },
                      }}`
